build.gradle

```diff
@@ -28,8 +29,27 @@ ext {
     javalin = '5.3.1'
 }
 
+dependencyCheck {
+    formats=['SARIF', 'JSON', 'HTML']
+    failBuildOnCVSS=5
+
+    analyzers {
+        assemblyEnabled = false
+        nugetconfEnabled = false
+        nodeEnabled = false
+    }
+}
+
+group = 'com.exactpro.th2'
+version = release_version
+String suffix = property('version_suffix').toString()
+if (!suffix.isEmpty()) {
+    version = "$version-$suffix"
+}
```

Comment:
Please, move this into `buildSrc/src/main/groovy/com.exactpro.th2.common-convensions.gradle`